src/business-logic/services/rundown-timeline-service.ts

```diff
@@ -31,6 +33,13 @@ export class RundownTimelineService implements RundownService {
   ) {}
 
   public async activateRundown(rundownId: string): Promise<void> {
+
+    const basicRundowns: BasicRundown[] = await this.rundownRepository.getBasicRundowns();
+    const activeRundown: BasicRundown | undefined = basicRundowns? basicRundowns.find(rundown => rundown.isActive()) : undefined;
```

Comment:
`this.rundownRepository.getBasicRundowns()` returns an array, so there is no need to do the undefined check on line 38.
You can just write: 
```
const activeRundown: BasicRundown | undefined = basicRundowns.find(rundown => rundown.isActive())
```